src/azure-cli/azure/cli/command_modules/resource/_help.py

```diff
@@ -2278,10 +2278,12 @@
 helps['tag create'] = """
 type: command
 short-summary: Create tags on a specific resource.
+note: az tag wipes existing tags. To solve this, one can get and buffer existing tags and add new ones to buffer. Then set buffered tags on resource.
```

Comment:
```suggestion
```

Reply:
https://github.com/Azure/azure-cli/pull/19057#discussion_r684063125 As I mentioned above, this line is no longer needed, so I deleted it~